buildenv/jenkins/perfPipeline.groovy

```diff
@@ -1,14 +1,19 @@
 #!groovy
-def testStats = [:]
-def baselineStats = [:]
-def testRuntimes = []
-def baselineRuntimes = []
+
 def testParams = []
 def baselineParams = []
-int PERF_ITERATIONS = params.PERF_ITERATIONS ? params.PERF_ITERATIONS.toInteger() : 4
 boolean RUN_BASELINE = (params.RUN_BASELINE != null) ? params.RUN_BASELINE.toBoolean() : true
+int PERF_ITERATIONS = params.PERF_ITERATIONS.toInteger()
```

Comment:
We no longer get PERF_ITERATIONS from the job parameter. It only gets set in `PERFCONFIG_JSON`.  `params.PERF_ITERATIONS.toInteger()` will cause job failure as params.PERF_ITERATIONS will be null.